### PR TITLE
Vide player page/history page

### PR DESCRIPTION
**Pull request by sanathkumar26700**
1. Video player page with video player

2. History page
     - Updates by adding videos played in video player
     - delete button to remove the particular video from the page
     - Clear history button removes all the videos

### CODE REVIEW

File: src/Components/Sidebar/sidebar.jsx [hunk 1]
```diff
@@ -6,11 +6,12 @@ import '../../Utilities/CSS/Utilities.css'
 function Sidebar() {
 
     const location = useLocation()
-
+    const path = location.pathname
```
Comment (msdeshmukh009) [inline src/Components/Sidebar/sidebar.jsx:9]: You can also write `const { pathname } = useLocation();`

File: src/Components/Video Card/video-card.jsx [hunk 2]
```diff
@@ -1,32 +1,46 @@
...
-        </div>
-        <div className="card__product--content">
-            <h2 className="card__product--title">{video.title}</h2>
-            <span className="card__product--category">{video.views} views</span>
-            <img className="avatar__round avatar--sm" src={video.channelImg} alt='channel logo'/>
-            <h4 className="card__product--company">{video.channelName} </h4>
-            <p className="card__product--description"> {video.description} </p>
+            <div onClick={() => navigate(`/video/${video.youtubeId}`)}>
+                <div className="card__product--img">
+                    <img className="img" src={video.thumbnailUrl} alt="video thumbnail"/>
```
Comment (msdeshmukh009) [inline src/Components/Video Card/video-card.jsx:19]: `alt = {video.title}` will be good for accessibility

**Review by msdeshmukh009 — APPROVED** [review]
History management is working as expected. Good to go.

File: src/Components/Sidebar/sidebar.jsx (hunk 1 above)
Comment (sanathkumar26700) [reply src/Components/Sidebar/sidebar.jsx:9]: yes, will change it.

**Pull request merged**